### ISSUE TITLE
OLH licence link contrast fail

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#5342]
<img width="838" height="341" alt="Image" src="https://github.com/user-attachments/assets/d0af17dc-945b-4db0-aac2-3bbf8570e3ba" />

OLH licence link text overflowing area, giving white text on white background.

**Comment by StephDriver** [comment]
The problem was `.has-tip` makes the license link `display: inline-block`, which sizes it to its content.
